share/rocm/cmake/ROCMSetupVersion.cmake

```diff
@@ -154,7 +154,18 @@ endfunction()
 
 function(rocm_set_soversion LIBRARY_TARGET SOVERSION)
     if(NOT WIN32)
-        rocm_version_regex_parse("^([0-9]+).*" SOVERSION_MAJOR "${SOVERSION}")
-        set_target_properties(${LIBRARY_TARGET} PROPERTIES VERSION ${SOVERSION} SOVERSION ${SOVERSION_MAJOR})
+        rocm_version_regex_parse("^([0-9]+).*" LIB_VERSION_MAJOR "${SOVERSION}")
+        rocm_version_regex_parse("^[0-9]+\\.([0-9]+).*" LIB_VERSION_MINOR "${SOVERSION}")
```

Comment:
This shouldn't chopp off the other versions added in the build.

Reply:
I have updated the changes not to chop off the other versions.

Reply:
Please review, I have updated the changes.